Podfile

```diff
@@ -90,7 +90,7 @@ target 'WooCommerce' do
 
   wordpress_shared
 
-  pod 'WordPressUI', '~> 1.13'
+  pod 'WordPressUI', :git => 'https://github.com/wordpress-mobile/WordPressUI-iOS.git', :branch => 'task/fix-crash-BottomSheetViewController'
```

Comment:
Please point this to the library's `trunk` after merging the fix.

Reply:
@itsmeichigo @pmusolino 1.13.1 has been shipped and this branch now points to it.

Reply:
Thank you both!